source/solvers/gls_nitsche_navier_stokes.cc

```diff
@@ -333,7 +353,6 @@ GLSNitscheNavierStokesSolver<dim, spacedim>::solve()
         }
 
       this->postprocess(false);
-      // postprocess_ht();
```

Comment:
remove if not needed

Reply:
When a statement is in red it means this is a line that has been removed. I cannot remove what has already been removed ^^!